sklearn/mixture/gmm.py

```diff
@@ -218,18 +220,23 @@ class GMM(BaseEstimator):
     >>> g.fit(20 * [[0]] +  20 * [[10]]) # doctest: +NORMALIZE_WHITESPACE
     GMM(covariance_type='diag', init_params='wmc', min_covar=0.001,
             n_components=2, n_init=1, n_iter=100, params='wmc',
-            random_state=None, thresh=0.01)
+            random_state=None, thresh=None, tol=0.001)
     >>> np.round(g.weights_, 2)
     array([ 0.5,  0.5])
 
     """
 
     def __init__(self, n_components=1, covariance_type='diag',
-                 random_state=None, thresh=1e-2, min_covar=1e-3,
+                 random_state=None, thresh=None, tol=1e-3, min_covar=1e-3,
                  n_iter=100, n_init=1, params='wmc', init_params='wmc'):
+        if thresh is not None:
+            warnings.warn("'thresh' was replaced by 'tol' and will "
```

Comment:
It's better to also state the version where it has first been deprecated:

"'thresh' has been replaced by 'tol' in 0.16 and will be removed in 0.18."